test/test-utils/string-utils.ts

```diff
@@ -68,7 +74,7 @@ export function getEnumStr<E extends EnumOrObject>(
     stringPart = casingFunc(stringPart);
   }
 
-  return `${prefix}${stringPart}${suffix} (=${val})`;
+  return `${prefix}${stringPart}${suffix}${omitValue ? ` (=${val})` : ""}`;
```

Comment:
Isn't this reversed? I would expect it to display nothing if `omitValue` is `true`, based on its name.
```suggestion
  return `${prefix}${stringPart}${suffix}${omitValue ? "" : ` (=${val})`}`;
```